### ISSUE TITLE
Needs a simple on/off button to disable add-on temporarily

### ISSUE THREAD

**Issue by nonjenec** [ColdSpirit0/FastCopyLinks#6]
I think going to add-ons menu, finding Fast Copy Link and disabling it is not really convenient. "why disabling them?" let's say one of my daily scenario involves copying some text that is also an url, i want said text in my clipboard but Firefox confuses them and copies the url instead. I could right click and "copy" instead but old habits die hard hehe. also thanks for this simple add-on!

**Comment by m0ck69** [comment]
Agreed. on/off button would be convenient.

**Comment by chapmanjacobd** [comment]
yeah I think if there was a browser API to check if any text is selected then there should be an option or default to not copy the hovered link
